src/HtmlGenerator/Program.cs

```diff
@@ -71,12 +94,27 @@ static void Main(string[] args)
             }
         }
 
+        private static string GetProjectPath(string arg)
+        {
+            var project = Path.GetFullPath(arg);
+            if (File.Exists(project))
+            {
+                if (project.EndsWith(".sln", StringComparison.OrdinalIgnoreCase) ||
+                    project.EndsWith(".csproj", StringComparison.OrdinalIgnoreCase) ||
+                    project.EndsWith(".vbproj", StringComparison.OrdinalIgnoreCase))

```

Comment:
We should extract this piece into a separate method if its duplicated, e.g. IsSupportedProject(string filePath)